examples/tutorial/01_Workflow_Introduction.ipynb

```diff
@@ -699,41 +699,32 @@
     "\n",
-    "You can also easily add widgets to control filtering, selection, and other options interactively, either here in the notebook, or in a standalone server by marking the servable objects with `.servable()` then running the .ipynb file through Bokeh Server or extracting the code to a separate .py file and doing the same thing:"
+    "You can also easily add widgets to control filtering, selection, and other options interactively, either here in the notebook or by putting the same code (plus `panel.servable();`) in a separate file and running it as a standalone server:"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {
-    "slideshow": {
-     "slide_type": "slide"
-    }
-   },
+   "metadata": {},
    "outputs": [],
    "source": [
-    "import param, panel as pn\n",
-    "from colorcet import palette\n",
+    "def make_view(location='dropoff', cmap=palette['fire'], alpha=1):\n",
+    "    pts = hv.Points(taxi, [location+'_x', location+'_y'])\n",
+    "    trips = datashade(pts, cmap=cmap)\n",
+    "    return tiles.options(alpha=alpha) * trips\n",
     "\n",
-    "class NYCTaxi(param.Parameterized):\n",
-    "    alpha = param.Magnitude(default=0.75, doc=\"Map tile opacity\")\n",
-    "    cmap = param.ObjectSelector('fire', objects=['fire','bgy','bgyw','bmy','gray','kbc'])\n",
-    "    location = param.ObjectSelector(default='dropoff', objects=['dropoff', 'pickup'])\n",
+    "colormaps = {n: palette[n] for n in ['fire','bgy','bgyw','bmy','gray','kbc']}\n",
     "\n",
-    "    def make_view(self, **kwargs):\n",
-    "        pts   = hv.Points(taxi, [self.location+'_x', self.location+'_y'])\n",
-    "        trips = datashade(pts, cmap=palette[self.cmap], **dopts)\n",
-    "        return tiles.options(alpha=self.alpha) * trips\n",
+    "explorer = pn.interact(make_view, cmap=colormaps, location=['dropoff', 'pickup'], alpha=(0, 1.))\n",
     "\n",
-    "explorer = NYCTaxi(name=\"Taxi explorer\")\n",
-    "pn.Row(explorer.param, explorer.make_view()).servable()\n"
+    "pn.Row(pn.Column('# NYC Taxi Explorer', explorer[0]), explorer[1]).servable()"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Note that in this simple app, the method `make_view` is called whenever *any* of the parameters change (alpha, colormap, or location), but you can get a more responsive interface if you take the time to declare which computations depend on which parameters (see the [Deploying Bokeh Apps tutorial](./13_Deploying_Bokeh_Apps.ipynb)).\n",
+    "Note that in this simple app, the method `make_view` is called whenever *any* of the parameters change (alpha, colormap, or location) triggering a full rerender, but you can get a more responsive interface if you take the time to clare which computations depend on which parameters (see the [Deploying Bokeh Apps tutorial](./13_Deploying_Bokeh_Apps.ipynb)).\n",
```

Comment:
```suggestion
    "Note that in this simple app, the method `make_view` is called whenever *any* of the parameters change (alpha, colormap, or location) triggering a full rerender, but you can get a more responsive interface if you take the time to declare which computations depend on which parameters (see the [Deploying Bokeh Apps tutorial](./13_Deploying_Bokeh_Apps.ipynb)).  Either way, the app should work the same here in the notebook (if you have a live Python process) or as a standalone server by calling `panel serve` with either the name of a Python file with the above code or simply the name of this notebook (where it will serve any objects marked `.servable()`).)"\n",
```

Reply:
(Not sure if anyone can simplify that mouthful of text!